api/src/test/java/org/openmrs/api/PatientServiceTest.java

```diff
@@ -2096,7 +2094,7 @@ public void savePatientIdentifier_shouldAllowLocationToBeNullWhenLocationBehavio
 	 */
 	@Test(expected = ValidationException.class)
 	public void savePatientIdentifier_shouldAllowLocationToBeNullWhenLocationBehaviourIsRequired() {
-		PatientIdentifier patientIdentifier = patientService.getPatientIdentifier(7);
+		PatientIdentifier patientIdentifier = patientService.getPatientIdentifier(9);
```

Comment:
This was a bad patient, they are voided but it worked before because they had a non-voided identifier which I changed to voided and it got broken, so I had to pick another non-voided patient with a non voided identifier